Assets/Plugins/Source/Editor/Utility/UnityPackageCreationUtility.cs

```diff
@@ -200,6 +227,41 @@ public static void CreateUPMTarball(string outputPath, string json_file)
             );
         }
 
+        public static void CreateUPMTarballV2(string preset)
```

Comment:
I think we need to clarify what makes it a "V2" Tarball. I think we had discussed either wholly replacing the old function with this new one, or making it clear through comments what the distinction was, and renaming this to be something more representative of what it does differently.